package.json

```diff
@@ -21,6 +21,7 @@
   ],
   "dependencies": {
     "axios": "^0.17.1",
+    "gcp-metadata": "^0.4.1",
```

Comment:
Leaving a reminder here to make sure this gets updated after the semver major changes in gcp-metadata land.

Reply:
Do you still need to update this?